tests/detect-bypass/test.rules

```diff
@@ -0,0 +1,3 @@
+alert tcp any any -> any any (bypass; content:"GET "; sid:1;)
+alert http any any -> any any (bypass; content:"message"; http_server_body; sid:2;)
+alert http any any -> any any (bypass; content:"message"; http_host; sid:3;)
```

Comment:
Since this is a new test, should we update the rule to use the new keywords `http.response_body` and `http.host`?

Reply:
indeed. @TheKharleeci we changed these keywords to dot notation a while ago, you can find the table of legacy content modifiers/sticky buffers and their corresponding new keyword here: https://suricata.readthedocs.io/en/latest/rules/http-keywords.html#http-keywords